tests/voq/test_fabric_cli_and_db.py

```diff
@@ -172,122 +177,127 @@ def test_fabric_cli_isolate_supervisor(duthosts, enum_supervisor_dut_hostname):
+                    cmd_output = duthost.shell(cmd, module_ignore_errors=True)["stdout"].split("\n")
+                    tokens = cmd_output[0].split()
+                    isolateStatus = tokens[0]
+                    pytest_assert(
+                        len(tokens) > 0,
+                        "FABRIC_PORT_TABLE:Fabric{} isolateStatus not found in APPL_DB, {} ".format(localPort,
+                                                                                                    asicName))
+                    pytest_assert(
+                        isolateStatus == "False",
+                        "Port {} APPL_DB initial isolateStatus is '{}', expected False".format(localPort,
+                                                                                               isolateStatus))
+
+                    # Isolate the port
+                    cmd = "sudo config fabric port isolate {} -n {}".format(localPort, asicName)
+                    cmd_output = duthost.shell(cmd, module_ignore_errors=True)
+                    stderr_output = cmd_output["stderr"]
+                    pytest_assert(
+                        len(stderr_output) <= 0,
+                        "command: {} failed, error: {}".format(cmd, stderr_output))
+
+                    # Check the isolateStatus in CONFIG_DB
+                    cmd = "sonic-db-cli -n {} CONFIG_DB hget 'FABRIC_PORT|Fabric{}' isolateStatus".format(asicName,
+                                                                                                          localPort)
+                    cmd_output = duthost.shell(cmd, module_ignore_errors=True)["stdout"].split("\n")
+                    tokens = cmd_output[0].split()
+                    isolateStatus = tokens[0]
+                    pytest_assert(len(tokens) > 0,
+                                  "FABRIC_PORT|Fabric{} isolateStatus not found in CONFIG_DB, {} ".format(localPort,
+                                                                                                          asicName))
+                    pytest_assert(
+                        isolateStatus == "True",
+                        "Port {} CONFIG_DB initial isolateStatus is '{}', expected True".format(localPort,
+                                                                                                isolateStatus))
+
+                    # Check the isolateStatus in APPL_DB
+                    cmd = "sonic-db-cli -n {} APPL_DB hget 'FABRIC_PORT_TABLE:Fabric{}' isolateStatus".format(asicName,
+                                                                                                              localPort)
+                    cmd_output = duthost.shell(cmd, module_ignore_errors=True)["stdout"].split("\n")
+                    tokens = cmd_output[0].split()
+                    isolateStatus = tokens[0]
+                    pytest_assert(
```

Comment:
Same comment as above. Please move `pytest_assert` to above accessing `tokens[0]`